Swift graduates from school. (#14)
apple/swift-book: Sources/TSPL/TSPL.docc/LanguageGuide/TheBasics.md

# The Basics

Swift is a programming language for iOS, macOS, watchOS, and tvOS app development.

> If you have prior programming experience in languages such as C or Objective-C,
> some parts of Swift will be familiar to you.

Reviewer: ```suggestion
If you have experience developing in C or Objective-C,
many parts of Swift will be familiar to you.
```

As noted in the forum post, content-level changes will be solicited at a later time. To keep this PR focused on the proposed change (deleting "new"), the most faithful rewording of the sentence that follows would be something like this.
